app/src/pages/Protocols/hooks/index.ts

```diff
@@ -182,6 +183,90 @@ export const useRequiredProtocolHardwareFromAnalysis = (
   }
 }
 
+/**
+ * Returns an array of RunTimeParameters objects that are optional by the given protocol ID.
+ *
+ * @param {string} protocolId The ID of the protocol for which required hardware is being retrieved.
+ * @returns {RunTimeParameters[]} An array of RunTimeParameters objects that are required by the given protocol ID.
+ */
+
+export const useRunTimeParameters = (
+  protocolId: string
+): RunTimeParameters[] => {
+  const { data: protocolData } = useProtocolQuery(protocolId)
+  const { data: analysis } = useProtocolAnalysisAsDocumentQuery(
+    protocolId,
+    last(protocolData?.data.analysisSummaries)?.id ?? null,
+    { enabled: protocolData != null }
+  )
+
+  const mockData: RunTimeParameters[] = [
+    {
+      displayName: 'Dry Run',
+      variableName: 'DRYRUN',
+      description: 'a dry run description',
+      default: false,
+    },
+    {
+      displayName: 'Use Gripper',
+      variableName: 'USE_GRIPPER',
+      description: '',
```

Comment:
I think if we can fill all description, it will be good for the further development. (Especially, ODD)
I think even `abc` or `123` would be fine.

```suggestion
      description: 'use gripper',
```

Reply:
oh ya good call, i'll add a variety of lengths to the description to help development

Reply:
never mind. I think I can add descriptions in my PR since I will use your PR 😁

Reply:
oh oops i just pushed the descriptions. i'll merge asap once CI clears!

Reply:
thanks!